flybywiresim/aircraft
# feat(efcs/ap/athr): improve support for active pause

## Summary of Changes
This PR improves support for active pause.

Before this PR enabling active pause didn't stop the fly-by-wire system to control the plane. Because the airplane wasn't moving, the control inputs didn't have any impact and increasingly large elevator and trim values have been achieved which caused problems when disabling active pause later.

In active pause, changes to AP modes etc. is possible and should be correctly reflected but no updates to the flight directory or flight controls will happen. This is done to ensure a smooth start when active pause is disabled.

## Testing instructions
* Use active pause in various situations
* Change AP modes during active pause
* Change ATHR state during active pause

When disabling no jumps in control of the airplane should happen.


## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page
